core/behavior/profile_manager.py

```
@@ -0,0 +1,104 @@
+        if not baseline.is_ready():
+            return None
+        return baseline.get_baseline()
+
+    def get_confidence(self, user_id: str, scope: str = "global") -> float:
+        key = (user_id, scope)
+        if key not in self._baselines:
+            return 0.0
+        return self._baselines[key].confidence
+
+    def prune_stale(self, min_messages: int = 5, max_age_seconds: float = 86400 * 30) -> int:
+        """清理低活跃用户（<5条消息且30天未活跃），返回清理数"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)